src/pages/lessons/fundamentals/eth-intro-part-2.mdx

```diff
@@ -0,0 +1,279 @@
+
+It’s hard not to collect accounts in today’s web. You have one for each social media app, news site, delivery service, retailer, and transit provider — to name a few. Each of these accounts lives on the company's servers, which makes you subject to their terms and conditions, privacy policy, and security practices. Your account can be frozen, deleted, censored, or altered at the discretion of the host company.
+
+Web3 represents a paradigm shift for account management: you and you alone own your Ethereum accounts. When you create an account, it’s done outside of the scope of any company and can travel with you to multiple apps. In fact, creating an Ethereum account doesn’t require interaction with the Ethereum blockchain at all. Let’s create one now to prove it.
+
+<Callout emoji="💡" size="md" variant="info">
+**Note**: This exercise is purely for educational purposes. Don't store real value in an account unless you understand the security implications. Some mistakes cannot be undone! More context to come.
+</Callout>
+
+## Creating an account
+
+Same drill as last time: the concepts will be demonstrated in an IPython shell. If you’re not a Python developer, no problem. Just follow along conceptually.
+
+### Environment setup
+
+Three steps to set the stage:
+
+1. Install web3.py, eth-tester, and IPython (if you didn't already in Part 1):
+
+```bash
+$ pip install web3 "web3[tester]" ipython
+```
+
+2. Start up a new sandbox:
+
+```bash
+$ ipython
+```
+
+3. Import the Web3 module:
+
+```python
+In [1]: from web3 import Web3
+```
+
+### Account generation
+
+Let's create that account:
+
+```python
+In [2]: w3 = Web3() # No provider necessary for this demo
+
+In [3]: acct = w3.eth.account.create()
+
+# public address:
+In [4]: acct.address
+Out[4]: '0x33736Bf0Ac7A046eAC36648ca852B91EAe5f567A'
+
+# private key:
+In [5]: acct.key
+Out[5]: HexBytes('0x7aca78f5e54...')
+```
+
+That's all there is to it! There's no registration process and no round-trip to the blockchain or any server. In fact, you can disconnect from the Internet altogether and still create a valid Ethereum account.
+
+In the code above, you'll find two components of an account: a public address and a private key. Put simply, a **private key** is the password for an account. A **public address** is a shareable account number derived from the private key. As seen in the code sample, both are typically represented as hexadecimal numbers.
+
+## Using an account
+
+This is an important takeaway: **the only way to affect a change on the blockchain is via a transaction** and every transaction must be signed by an account. Accounts can initiate transactions which transfer ether, deploy smart contracts, or interact with contracts to do things like mint new tokens. Let's briefly explore each.
+
+### Transferring ether
+
+Recall that `EthereumTesterProvider` enables a test environment seeded with accounts and fake ether. Let's start by viewing some test accounts and an account balance:
```

Comment:
Do you want to make this a question by replacing with a question mark at the end instead of full stop? It will personalise it for the learner a lot :)